<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NumKeyBoard</class>
 <widget class="QWidget" name="NumKeyBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NumKeyBoard</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#NumKeyBoard
{
	
	border-image: url(:/Image/Resources/Image/img_bg.png);
}
QToolButton
{
min-height:80px;
min-width:80px;
max-height:80px;
max-width:80px;
	font: 20pt &quot;微软雅黑&quot;;
}
QLCDNumber
{
min-height:100px;
min-width:100px;
max-height:100px;
max-width:100px;
font: 20pt &quot;微软雅黑&quot;;
background-color: rgb(255, 255, 255);
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_title" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>95</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>95</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>25</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>25</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btn_back">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">#btn_back
{
min-width:55px;
max-width:55px;
min-height:50px;
max-height:50px;
border-radius: 10px;
}
#btn_back{border-image: url(:/Image/Resources/Image/btn_Return_nor.png);}
#btn_back:hover{border-image:url(:/Image/Resources/Image/btn_Return_sele.png);}
#btn_back:pressed{border-image:url(:/Image/Resources/Image/btn_Return_press.png);}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_title">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
min-width:379px;
max-width:379px;
min-height:71px;
max-height:71px;
border-image: url(:/Image/Resources/Image/请输入取件码.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_home">
        <property name="styleSheet">
         <string notr="true">#btn_home
{
min-width:55px;
max-width:55px;
min-height:50px;
max-height:50px;
border-radius: 10px;
}
#btn_home{border-image: url(:/Image/Resources/Image/btn_home_nor.png);}
#btn_home:hover{border-image:url(:/Image/Resources/Image/btn_home_sele.png);}
#btn_home:pressed{border-image:url(:/Image/Resources/Image/btn_home_press.png);}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_input" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>200</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>35</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="display">
          <property name="maximumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>20</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_1">
          <property name="styleSheet">
           <string notr="true">
#label_1
{
min-width:85px;
max-width:85px;
min-height:94px;
max-height:94px;
border-radius: 12;
color: rgb(122, 118, 117);
	font: 64px &quot;黑体&quot;;
}
#label_1{border-image:url(:/Image/Resources/Image/label_Inputbox_sele.png)}</string>
          </property>
          <property name="text">
           <string>-</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">
#label_2
{
min-width:85px;
max-width:85px;
min-height:94px;
max-height:94px;
border-radius: 12;
color: rgb(122, 118, 117);
	font: 64px &quot;黑体&quot;;
}
#label_2{border-image:url(:/Image/Resources/Image/label_Inputbox_sele.png)}</string>
          </property>
          <property name="text">
           <string>-</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">
#label_3
{
min-width:85px;
max-width:85px;
min-height:94px;
max-height:94px;
border-radius: 12;
color: rgb(122, 118, 117);
	font: 64px &quot;黑体&quot;;
}
#label_3{border-image:url(:/Image/Resources/Image/label_Inputbox_sele.png)}</string>
          </property>
          <property name="text">
           <string>-</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">
#label_4
{
min-width:85px;
max-width:85px;
min-height:94px;
max-height:94px;
border-radius: 12;
color: rgb(122, 118, 117);
	font: 64px &quot;黑体&quot;;
}
#label_4{border-image:url(:/Image/Resources/Image/label_Inputbox_sele.png)}</string>
          </property>
          <property name="text">
           <string>-</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">
#label_5
{
min-width:85px;
max-width:85px;
min-height:94px;
max-height:94px;
border-radius: 12;
color: rgb(122, 118, 117);
	font: 64px &quot;黑体&quot;;
}
#label_5{border-image:url(:/Image/Resources/Image/label_Inputbox_sele.png)}</string>
          </property>
          <property name="text">
           <string>-</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="styleSheet">
           <string notr="true">
#label_6
{
min-width:85px;
max-width:85px;
min-height:94px;
max-height:94px;
border-radius: 12;
color: rgb(122, 118, 117);
	font: 64px &quot;黑体&quot;;
}
#label_6{border-image:url(:/Image/Resources/Image/label_Inputbox_sele.png)}</string>
          </property>
          <property name="text">
           <string>-</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_num" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>35</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>35</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="horizontalSpacing">
          <number>60</number>
         </property>
         <property name="verticalSpacing">
          <number>10</number>
         </property>
         <item row="0" column="0">
          <widget class="QToolButton" name="toolButton_1">
           <property name="minimumSize">
            <size>
             <width>108</width>
             <height>108</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>108</width>
             <height>108</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#toolButton_1
{
min-width:108px;
max-width:108px;
min-height:108px;
max-height:108px;
border-radius: 10px;
color: rgba(255, 255, 255,0);
}
#toolButton_1{border-image: url(:/Image/Resources/Image/1_nor.png);}
#toolButton_1:hover{border-image:url(:/Image/Resources/Image/1_nor.png);}
#toolButton_1:pressed{border-image:url(:/Image/Resources/Image/1_press.png);}</string>
           </property>
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QToolButton" name="toolButton_2">
           <property name="minimumSize">
            <size>
             <width>108</width>
             <height>108</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>108</width>
             <height>108</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#toolButton_2
{
min-width:108px;
max-width:108px;
min-height:108px;
max-height:108px;
border-radius: 10px;
color: rgba(255, 255, 255,0);
}
#toolButton_2{border-image: url(:/Image/Resources/Image/2_nor.png);}
#toolButton_2:hover{border-image:url(:/Image/Resources/Image/2_nor.png);}
#toolButton_2:pressed{border-image:url(:/Image/Resources/Image/2_press.png);}</string>
           </property>
           <property name="text">
            <string>2</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QToolButton" name="toolButton_4">
           <property name="minimumSize">
            <size>
             <width>108</width>
             <height>108</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>108</width>
             <height>108</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#toolButton_4
{
min-width:108px;
max-width:108px;
min-height:108px;
max-height:108px;
border-radius: 10px;
color: rgba(255, 255, 255,0);
}
#toolButton_4{border-image: url(:/Image/Resources/Image/4_nor.png);}
#toolButton_4:hover{border-image:url(:/Image/Resources/Image/4_nor.png);}
#toolButton_4:pressed{border-image:url(:/Image/Resources/Image/4_press.png);}</string>
           </property>
           <property name="text">
            <string>4</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QToolButton" name="toolButton_5">
           <property name="minimumSize">
            <size>
             <width>108</width>
             <height>108</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>108</width>
             <height>108</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#toolButton_5
{
min-width:108px;
max-width:108px;
min-height:108px;
max-height:108px;
border-radius: 10px;
color: rgba(255, 255, 255,0);
}
#toolButton_5{border-image: url(:/Image/Resources/Image/5_nor.png);}
#toolButton_5:hover{border-image:url(:/Image/Resources/Image/5_nor.png);}
#toolButton_5:pressed{border-image:url(:/Image/Resources/Image/5_press.png);}</string>
           </property>
           <property name="text">
            <string>5</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QToolButton" name="toolButton_6">
           <property name="minimumSize">
            <size>
             <width>108</width>
             <height>108</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>108</width>
             <height>108</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#toolButton_6
{
min-width:108px;
max-width:108px;
min-height:108px;
max-height:108px;
border-radius: 10px;
color: rgba(255, 255, 255,0);
}
#toolButton_6{border-image: url(:/Image/Resources/Image/6_nor.png);}
#toolButton_6:hover{border-image:url(:/Image/Resources/Image/6_nor.png);}
#toolButton_6:pressed{border-image:url(:/Image/Resources/Image/6_press.png);}</string>
           </property>
           <property name="text">
            <string>6</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QToolButton" name="toolButton_7">
           <property name="minimumSize">
            <size>
             <width>108</width>
             <height>108</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>108</width>
             <height>108</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#toolButton_7
{
min-width:108px;
max-width:108px;
min-height:108px;
max-height:108px;
border-radius: 10px;
color: rgba(255, 255, 255,0);
}
#toolButton_7{border-image: url(:/Image/Resources/Image/7_nor.png);}
#toolButton_7:hover{border-image:url(:/Image/Resources/Image/7_nor.png);}
#toolButton_7:pressed{border-image:url(:/Image/Resources/Image/7_press.png);}</string>
           </property>
           <property name="text">
            <string>7</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QToolButton" name="toolButton_8">
           <property name="minimumSize">
            <size>
             <width>108</width>
             <height>108</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>108</width>
             <height>108</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#toolButton_8
{
min-width:108px;
max-width:108px;
min-height:108px;
max-height:108px;
border-radius: 10px;
color: rgba(255, 255, 255,0);
}
#toolButton_8{border-image: url(:/Image/Resources/Image/8_nor.png);}
#toolButton_8:hover{border-image:url(:/Image/Resources/Image/8_nor.png);}
#toolButton_8:pressed{border-image:url(:/Image/Resources/Image/8_press.png);}</string>
           </property>
           <property name="text">
            <string>8</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QToolButton" name="toolButton_9">
           <property name="minimumSize">
            <size>
             <width>108</width>
             <height>108</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>108</width>
             <height>108</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#toolButton_9
{
min-width:108px;
max-width:108px;
min-height:108px;
max-height:108px;
border-radius: 10px;
color: rgba(255, 255, 255,0);
}
#toolButton_9{border-image: url(:/Image/Resources/Image/9_nor.png);}
#toolButton_9:hover{border-image:url(:/Image/Resources/Image/9_nor.png);}
#toolButton_9:pressed{border-image:url(:/Image/Resources/Image/9_press.png);}</string>
           </property>
           <property name="text">
            <string>9</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QToolButton" name="toolButton_backspace">
           <property name="minimumSize">
            <size>
             <width>108</width>
             <height>108</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>108</width>
             <height>108</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#toolButton_backspace
{
min-width:108px;
max-width:108px;
min-height:108px;
max-height:108px;
border-radius: 10px;
}
#toolButton_backspace{border-image: url(:/Image/Resources/Image/btn_keyboard2_nor.png);}
#toolButton_backspace:hover{border-image:url(:/Image/Resources/Image/btn_keyboard2_nor.png);}
#toolButton_backspace:pressed{border-image:url(:/Image/Resources/Image/btn_keyboard2_sele.png);}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QToolButton" name="toolButton_0">
           <property name="minimumSize">
            <size>
             <width>108</width>
             <height>108</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>108</width>
             <height>108</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#toolButton_0
{
min-width:108px;
max-width:108px;
min-height:108px;
max-height:108px;
border-radius: 10px;
color: rgba(255, 255, 255,0);
}
#toolButton_0{border-image: url(:/Image/Resources/Image/0_nor.png);}
#toolButton_0:hover{border-image:url(:/Image/Resources/Image/0_nor.png);}
#toolButton_0:pressed{border-image:url(:/Image/Resources/Image/0_press.png);}</string>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QToolButton" name="toolButton_enter">
           <property name="minimumSize">
            <size>
             <width>108</width>
             <height>108</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>108</width>
             <height>108</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#toolButton_enter
{
min-width:108px;
max-width:108px;
min-height:108px;
max-height:108px;
border-radius: 10px;
}
#toolButton_enter{border-image: url(:/Image/Resources/Image/btn_keyboard3_nor.png);}
#toolButton_enter:hover{border-image:url(:/Image/Resources/Image/btn_keyboard3_nor.png);}
#toolButton_enter:pressed{border-image:url(:/Image/Resources/Image/btn_keyboard3_sele.png);}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QToolButton" name="toolButton_3">
           <property name="minimumSize">
            <size>
             <width>108</width>
             <height>108</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>108</width>
             <height>108</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#toolButton_3
{
min-width:108px;
max-width:108px;
min-height:108px;
max-height:108px;
border-radius: 10px;
color: rgba(255, 255, 255,0);
}
#toolButton_3{border-image: url(:/Image/Resources/Image/3_nor.png);}
#toolButton_3:hover{border-image:url(:/Image/Resources/Image/3_nor.png);}
#toolButton_3:pressed{border-image:url(:/Image/Resources/Image/3_press.png);}</string>
           </property>
           <property name="text">
            <string>3</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtControl.qrc"/>
 </resources>
 <connections/>
</ui>
